app/views/users/show.html.erb

```diff
@@ -1,2 +1,31 @@
-<h1>Users</h1>
-<p>Show user details</p>
+<main>
+<section class="users">
+<div class="userdetails">
+  <img class="image" src="<%=@user.photo%>" alt="user_image" />
+  <div class="details">
+    <h2><%= @user.name%> </h2>
+    <p class="post">Number of posts: <%= @user.posts_counter%> </p>
+  </div> 
+</div
+</section>
+<section class="bio">
+  <h3>Bio</h3>
+  <p><%= @user.bio%> </p>
+</section>
+<section class="userpost">
+  <% @user.recent_posts.each do |post| %>
+    <%= link_to user_post_path(@user, id: post.id) do %>
+  <div class="posts">
+    <h4><%= post.title %></h4>
+    <p><%= post.text %> </p>
+    <span>
+      Comment: <%= post.comments_counter%>, Likes: <%= post.likes_counter || 0 %>
+    </span>
+  </div>
+  <% end %>
+  <% end %>
+</section>
```

Comment:
- [ ] Please show the post number on this page as it is available on the wireframe `post #1`. Your view is missing that.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/175052466-9e47f8fa-fe31-4586-bc31-9a08fde07e32.png)